flycheck.el

```diff
@@ -295,6 +295,17 @@ respectively."
   :safe #'flycheck-symbol-list-p)
 (make-variable-buffer-local 'flycheck-disabled-checkers)
 
+(defcustom flycheck-enabled-checkers nil
```

Comment:
I don't think that this should be a customizable variable, should it?  To me it's internal state only.  I'd turn it into a `defvar-local` and move it down beneath the definition of `flycheck-may-enable-checker`.